tests/gen/Generated.hack

```diff
@@ -481,31 +481,16 @@ final class Query extends \Slack\GraphQL\Types\ObjectType {
           ErrorTest::nonNullable(),
           async ($parent, $args, $vars) ==> \ErrorTestObj::getNonNullable(),
         );
-      case 'getConcrete':
```

Comment:
Hmm. Looks like we should make sort order here stable as well.

Reply:
Good catch; I added a new commit here that does that. I only sorted fields generated from methods (object/interface/root types). The other cases (enums, input objects, shape-backed object types) have a clear natural order so it's probably better to preserve it.